sgr.go

```diff
@@ -80,23 +81,28 @@ func handleSgr(p *ansi.Parser) (string, error) { //nolint:unparam
 		case 29:
 			str += "No crossed-out"
 		case 30, 31, 32, 33, 34, 35, 36, 37:
-			str += fmt.Sprintf("Foreground color: %s", basicColors[param.Param(0)-30])
+			str += fmt.Sprintf("ANSI foreground color: %s", basicColors[param.Param(0)-30])
 		case 38:
-			str += fmt.Sprintf("Foreground color: %d", readColor(&i, params))
+			c := readColor(&i, params)
+			str += fmt.Sprintf("%s foreground color: %s", getColorType(c), getColorLabel(c))
 		case 39:
 			str += "Default foreground color"
 		case 40, 41, 42, 43, 44, 45, 46, 47:
-			str += fmt.Sprintf("Background color: %s", basicColors[param.Param(0)-40])
+			str += fmt.Sprintf("ANSI background color: %s", basicColors[param.Param(0)-40])
 		case 48:
-			str += fmt.Sprintf("Background color: %d", readColor(&i, params))
+			c := readColor(&i, params)
+			str += fmt.Sprintf("%s background color: %s", getColorType(c), getColorLabel(c))
 		case 49:
 			str += "Default background color"
-		case 58, 59:
-			str += fmt.Sprintf("Underline color: %d", readColor(&i, params))
+		case 58:
+			c := readColor(&i, params)
+			str += fmt.Sprintf("%s underline color: %s", getColorType(c), getColorLabel(c))
+		case 59:
+			str += "Default underline color"
 		case 90, 91, 92, 93, 94, 95, 96, 97:
-			str += fmt.Sprintf("Bright foreground color: %s", basicColors[param.Param(0)-90])
+			str += fmt.Sprintf("Bright ANSI foreground color: %s", basicColors[param.Param(0)-90])
```

Comment:
i would maybe change this 
```suggestion
			str += fmt.Sprintf("ANSI foreground color: Bright %s", basicColors[param.Param(0)-90])
```